rest/taskana-rest-spring/src/main/java/pro/taskana/rest/WorkbasketAccessItemController.java

```diff
@@ -86,6 +100,36 @@
         return new ResponseEntity<>(pagedResources, HttpStatus.OK);
     }
 
+    /**
+     * This DELETE method delete all workbasketAccessItems that correspond the given accessId.
+     *
+     * @param accessId which need remove his workbasketAccessItems.
+     * @return ResponseEntity if the user is not authorized.
+     * @throws NotAuthorizedException if the user is not authorized.
+     */
+    @DeleteMapping("{accessId}")
```

Comment:
I understand that this is not scope of this PR, bur can we change the endpoint to "/v1/workbasket-access-items"? We should try to be consistent with out naming and the resource we are getting back is a WorkbasketAccessItem, isn't it?

Reply:
It's true. I should be careful with the consistency of the names. 
Thank you!